package.json

```
@@ -37,7 +37,8 @@
     "e2e:run": "wait-on http://localhost:6006 && cypress run -c video=true",
     "e2e:serve": "NODE_ENV=e2e yarn start --ci --quiet",
     "snapshot": "CYPRESS_updateSnapshots=true yarn e2e",
-    "release": "HUSKY=0 standard-version"
+    "release": "HUSKY=0 standard-version",
+    "check-bundle-size": "yarn build && yarn run bundlewatch --config ./.bundlewatchrc.json"
```

[Minor] I think can be just:

```suggestion
    "check-bundle-size": "yarn build && yarn run bundlewatch --config .bundlewatchrc.json"
```

---

Good point. Meant to double check that this also works then forgot about it.